### ISSUE TITLE
ibm-combo-box with (search) attribute has a violation for Accessibility

### ISSUE THREAD

**Issue by GSSCDylan** [IBM/carbon-components-angular#1590]
## Detailed description

> Describe in detail the issue you're having.

- **ibm-combo-box with (search) attribute will generate an input as following:**
` <input type="text" role="searchbox"   autocomplete="list">`
-  **But the 'autocomplete' attribute's token "list" is not appropriate for the input form field.**

> Is this a feature request (new component, new icon), a bug, or a general issue?

- **bug**

> Is this issue related to a specific component?

- **ibm-combo-box**

> What version of the Carbon Design System are you using?
`"carbon-components": "10.20.0", "carbon-components-angular": "4.17.0"`

> What browser are you working in?
Firefox
## Steps to reproduce the issue

1. Use browser to open an webpage that contains ibm-combo-box with (search) attribute
2. Run IBM Equal Access Accessibility Checker

## Additional information

* Screenshots or code
<img width="1404" alt="image" src="https://user-images.githubusercontent.com/73825731/97976561-d14baf00-1e05-11eb-9b19-ec2e6da714fe.png">
<img width="976" alt="image" src="https://user-images.githubusercontent.com/73825731/97977433-2cca6c80-1e07-11eb-9fd2-4faabe25c668.png">

**Comment by GSSCDylan** [comment]
Hi @cal-smith , please also help to fix this issue ASAP, it's also impact our AD AVT compliance. @stanislavgeorgiev  FYI
